spring-mvc-simple-2/src/main/java/com/baeldung/flash_attributes/controllers/PoemSubmission.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.flash_attributes.controllers;
+
+import com.baeldung.flash_attributes.model.Poem;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.servlet.support.RequestContextUtils;
+import org.springframework.web.servlet.view.RedirectView;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class PoemSubmission {
+
+    @GetMapping("/poem/success")
+    public String getSuccess(HttpServletRequest request) {
+        java.util.Map<String, ?> inputFlashMap = RequestContextUtils.getInputFlashMap(request);
```

Comment:
Couldn't we just import java.util.Map so that we don't have to fully qualify it here?

Reply:
Addressed